h/assets.py

```diff
@@ -112,7 +39,14 @@ def asset_response_subscriber(event):
 
 
 def includeme(config):
-    config.registry.settings.setdefault('webassets.bundles', 'h:assets.yaml')
+    WEBASSETS_DEFAULTS = {
```

Comment:
This would probably be better as simply:

```python
settings = config.registry.settings
settings.setdefault('webassets.bundles', 'h:assets.yaml')
settings.setdefault('webassets.base_dir', 'h:../build')
settings.setdefault('webassets.base_url', '/assets')
```